.gitignore

```diff
@@ -10,3 +10,4 @@ logging.cfg
 .*.swo
 apache/testapp.conf
 tmp/*
+forge_*.log
```

Comment:
if you don't specify the log directory as env variable, logs are created in root...this avoids adding them to git.